libcomposefs/lcfs-writer.c

```diff
@@ -1009,16 +1009,28 @@ void lcfs_node_set_gid(struct lcfs_node_s *node, uint32_t gid)
 	node->inode.st_gid = gid;
 }
 
+// Deprecated: this truncates rdev to 32 bits
```

Comment:
`__attribute__((deprecated))`?

Reply:
Done and that also helped me notice that I forgot to port a consumer of the API here, thanks!